### PR TITLE
24-BUNSPACE2024-Add-Standard-Text-On-Lore-Page

### LINKED ISSUE #24: Add the standard text on lore page
Add text to the lore page, standard text. Dont worry about styling. Just put it and center it to the center of the page

### PR DESCRIPTION
**Pull request by GrpFruit**
Fix lore text to the middle

## Extra Resources

- [The 10 commandments of navigating code reviews](https://techbeacon.com/app-dev-testing/10-commandments-navigating-code-reviews)
- [Google's Code Review Developer Guide](https://google.github.io/eng-practices/review/)

Closes #24

### CODE REVIEW

File: ui/src/pages/Lore/Lore.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 export function Lore() {
   return (
-    <h2>THIS IS LORE PAGE</h2>
+    <div className="lore">
+      <h2>THIS IS LORE PAGE</h2>
```
Comment (jmorales8) [inline ui/src/pages/Lore/Lore.tsx:4]: For this, create a bunch of text. Whether it be the anya text we use all over or somewhere else. I want to see what this looks like when we have a bunch of text

**Comment by jmorales8** [comment]
For this ticket, just put random nonsense and have them seperated by labels. 
Sort of like actual mui docs. Like location: blah blah blah,  margin to space it out. Character: blah blah blah. Message me more of thise if you get stuck

**Comment by GrpFruit** [comment]
@jmorales8 let me know how you'd like me to style this. As is it is really bare and not fitting to the StarWars intro format

**Comment by jmorales8** [comment]
> @jmorales8 let me know how you'd like me to style this. As is it is really bare and not fitting to the StarWars intro format

This ticket's sole purpose is to add standard text. There is another ticket to accomplish the starwars effect

**Comment by GrpFruit** [comment]
Does it look good as is or should I change it back to where the labels are above the top left of the text

File: ui/src/pages/Lore/lore.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  }
+}
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin-bottom: 0;
+}
```
Comment (jmorales8) [inline ui/src/pages/Lore/lore.scss:17]: For these, dont do the stylings like this. Add classnames, this is bad practice

**Comment by jmorales8** [comment]
> Does it look good as is or should I change it back to where the labels are above the top left of the text

This is fine. I like it like this. We do later updates on visuals but the structure is good.

File: ui/src/pages/Lore/lore.scss (hunk 2 above)
Comment (GrpFruit) [reply ui/src/pages/Lore/lore.scss:17]: this has been done lol xD

**Comment by jmorales8** [comment]
+1

**Pull request merged**